en/flight_modes/mission.md

```diff
@@ -46,8 +50,8 @@ For more information about mission planning, see:
 
 ## Mission Parameters
 
-Mission behaviour is affected by a number of parameters. These cover, for example, how the vehicle will behave if it looses connection to its remote control during a mission ([NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT)), the fixed-wing loiter radius ([NAV_LOITER_RAD](../advanced_config/parameter_reference.md#NAV_LOITER_RAD)), acceptance radius for reaching a waypoint etc.
+Mission behaviour is affected by a number of parameters. These cover, for example, how the vehicle will behave if it looses connection to its remote control during a mission ([NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT)), how the vehicle will be the fixed-wing loiter radius ([NAV_LOITER_RAD](../advanced_config/parameter_reference.md#NAV_LOITER_RAD)), acceptance radius for reaching a waypoint etc.
 
 These are documented here: [Parameter Reference > Mission](../advanced_config/parameter_reference.md#mission)
 
-
+> **Tip** You can also directly configure the most common failsafe actions e.g. what to do in case of low battery, RC loss, Data link loss (between vehicle and QGC) and geofence breach in the QGC's [**Vehicle Setup > Safety**](https://docs.qgroundcontrol.com/en/SetupView/Safety.html). Additionally, the **Safety** tab provides the commonly used Return to Home and Land Mode Settings which are also important for mission mode. Configuring these settings is an easier alternative to changing the corresponding parameters in the **Parameters** tab.

```

Comment:
Arguably this section could be copied into every single page, and I don't think that makes sense. So we should remove this, and possibly just link to info in a standalone failsafe page. Capture that in https://github.com/PX4/px4_user_guide/issues/205#issuecomment-382930174